vue/test/components/dropdown/dropdown.test.js

```diff
@@ -0,0 +1,163 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Dropdown", () => {
+    it("should show a message if there are no items defined", () => {
+        const items = [];
+        const messageEmpty = "No items";
+        const component = base.getComponent("Dropdown", {
+            props: { items: items, messageEmpty: messageEmpty }
+        });
+
+        const dropdownItems = component.findAll(".dropdown-item").length;
+        const messageEmptyItem = component.find(".dropdown-item-empty");
+        assert.strictEqual(dropdownItems, 0);
+        assert.strictEqual(messageEmptyItem.text(), "No items");
+    });
+
+    it("should have items accordingly with props", () => {
+        const items = [
+            {
+                value: "text_1",
+                label: "Text 1"
+            },
+            {
+                value: "text_2"
+            }
+        ];
+        const component = base.getComponent("Dropdown", {
+            props: { items: items }
+        });
+
+        const dropdownItems = component.findAll(".dropdown-item");
+        const firstItem = dropdownItems.at(0);
+        const secondItem = dropdownItems.at(1);
+        assert.strictEqual(dropdownItems.length, 2);
+        assert.strictEqual(firstItem.text(), "Text 1");
+        assert.strictEqual(secondItem.text(), "text_2");
+        assert.strictEqual(component.vm.$props.items, items);

```

Comment:
No need for this line because it's not testing internal behavior. What's the use to know that a prop was passed? The only way for that to fail was that if `base.getComponent` was bugged.